numpydoc/docscrape.py

```diff
@@ -555,7 +555,16 @@ def __str__(self):
 
 class ClassDoc(NumpyDocString):
 
-    extra_public_methods = ['__call__']
+    extra_public_methods = ['_call',
```

Comment:
Actually we can remove `_call`, too, since most operators don't provide useful information there anyway.